pkg/storing/chats/chats.go

```diff
@@ -0,0 +1,85 @@
+package chats
+
+import (
+	"github.com/jameycribbs/hare"
+	"github.com/jameycribbs/hare/datastores/disk"
+	"github.com/pkg/errors"
+	"log"
+	"nodemon/pkg/entities"
+	"nodemon/pkg/storing/common"
+)
+
+const (
+	chatsTableName = "chats"
+)
+
+type chat struct {
+	ID   int `json:"id"`
+	Chat entities.Chat
+}
+
+func (c *chat) GetID() int {
+	return c.ID
+}
+
+func (c *chat) SetID(id int) {
+	c.ID = id
+}
+
+// AfterFind required by Hare function
+func (c *chat) AfterFind(_ *hare.Database) error {
+	return nil
+}
+
+type Storage struct {
+	db *hare.Database
+}
+
+func NewStorage(path string) (*Storage, error) {
```

Comment:
Is this storage usable by other than telegram bot? If not, have to be moved to `tg_bot/internal`

Reply:
It is not yet. But I was going to use one `chats` table for both bots. So this `storage` is going to be common for both